app/assets/stylesheets/layout.scss

```diff
@@ -895,3 +895,23 @@ hr {
 .flash.flash-error p:first-of-type {
   margin-top: 0;
 }
+
+dl.form.errored.primer-new {
```

Comment:
Without the new class, our stye get's overridden by Primer. In CSS, if two selectors have equal specificity, the latest one gets used, and I guess application.scss is applied before imports. Adding a new class increases our specificity.

http://www.htmldog.com/guides/css/intermediate/specificity/

I've moved the stuff to `forms.scss`